DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchNearColliderLogic.cs

```diff
@@ -0,0 +1,161 @@
+﻿using HarmonyLib;
+using System;
+using UnityEngine;
+using Patch = GalacticScale.Scripts.PatchPlanetSize.PatchForPlanetSize;
+
+namespace GalacticScale.Scripts.PatchPlanetSize
+{
+    [HarmonyPatch(typeof(NearColliderLogic))]
+    static class PatchNearColliderLogic
+    {
+        [HarmonyPrefix]
+        [HarmonyPatch("GetVeinsInAreaNonAlloc")]
+        public static bool PatchGetVeinsInAreaNonAlloc(ref int __result, Vector3 center, float areaRadius, int[] veinIds, ref int ___activeColHashCount, ref int[] ___activeColHashes, ref ColliderContainer[] ___colChunks)
+        {
```

Comment:
Comment the part you changed. 

Plus the naming is bad just call the function : GetVeinsInAreaNonAlloc --> we already know it's a patch :) 
--> precise prefix or postfix in the name only if you patch the functio ntwice